### ISSUE TITLE
Changing highlighter colour to clear breaks link

### ISSUE THREAD

**Issue by hidara2000** [remnoteio/remnote-issues#1236]
**Describe the bug**
I hope to reference many topics in a pdf without covering the document in highlighter marks. I can change the colour of the highlighter to any colour but clear/white. When I select the last white box I lose my link to the pdf passage
![image](https://user-images.githubusercontent.com/15170494/120627813-6485b980-c4a7-11eb-952c-135e18bd6ed3.png)

**To Reproduce**
1. Go to an imported PDF
2. highlight on any word/sentence to reference
3. Press ctrl-c
4. Change highlight colour to any colour but the white box
5. Everything works and clicking on the link brings you to the relevant passage in the pdf
6. Select the the white box highlighter colour
5. Link to passage breaks

**Expected behavior**
Selecting the white box makes the highlighter mark invisible but functionality remains the same as when any other colour is selected

**Screenshots**
green highlighter working
![image](https://user-images.githubusercontent.com/15170494/120626255-d4934000-c4a5-11eb-9f99-18938906f1de.png)
Clear/white colour selected -> Link breaks
![image](https://user-images.githubusercontent.com/15170494/120626980-8d597f00-c4a6-11eb-8dd6-69aeb19c1ca4.png)

**Desktop (please complete the following information):**
 - OS: win 10
 - Browser Chrome
 - Version Version 90.0.4430.212 (Official Build) (64-bit)

**Additional context**
Cheers for a great app :)

**Comment by hannesfrank** [comment]
Thanks for the very clear report @hidara2000, the screenshots help a lot!

As it is currently implemented the last "highlight color" is actually a button to remove the highlight. 
This is the same mechanic as the normal text highlights (Ctrl + Alt + 4..9 are the highlight colors and Ctrl + Alt + 0 - "white" - removes the highlight) so it is working as intended.
I agree this could be more obvious. I'll add it to the backlog of design considerations for the UX rework.

Just referencing sections without or with less prominent visual indication sounds like a useful feature though. Could you please add this as a feature request to the forum? Maybe we could also add a button to toggle the highlights?

**Comment by hidara2000** [comment]
Cheers for the update, I suspected this was the case but hoped it wasn't ;)
I'll post that in the feature request.

In the meantime, i'd appreciate any tips on whether changing css settings I could hijack an existing highlighter colour and set it to white/clear?

I've tried to add transparancy
.highlight-color--red {
        background-color: #ffa5a700;
        padding: 0px 5px;
        color: ;
        border-radius: 2px;
    }

And to change it to white

.highlight-color--red {
        background-color: #ffffff;
        padding: 0px 5px;
        color: ;
        border-radius: 2px;
    }

But could get neither to work. Amy tips would be appreciated!
